### PR TITLE
VTCCA-3135 Valuation Summary move

### PR DESCRIPTION
**Pull request by darrell-plant**

### CODE REVIEW

File: app/controllers/Assessments.scala [hunk 1]
```diff
@@ -95,12 +96,16 @@ class Assessments @Inject()(
         logger.warn("property link assessment call failed", e)
         val linkF = if (owner) propertyLinks.getMyOrganisationPropertyLink(submissionId) else propertyLinks.getMyClientsPropertyLink(submissionId)
         linkF.map {
-          case Some(link) => Redirect(routes.Assessments.viewSummary(link.uarn, true))
+          case Some(link) => Redirect(getViewSummaryCall(link.uarn, pending = true, owner))
```
Comment (DanielConnelly) [inline app/controllers/Assessments.scala]: is there a reason we are not using reverse router?

**Comment by DanielConnelly** [comment]
LGTM 👍

File: app/controllers/Assessments.scala (hunk 1 above)
Comment (darrell-plant) [reply app/controllers/Assessments.scala]: we are using the reverse router

Comment (DanielConnelly) [reply app/controllers/Assessments.scala]: coolio.

**Comment by reddy-yattapu** [comment]
:+1:

**Pull request merged**